troubleshooting/current-activity-lag.sql

```diff
@@ -1,19 +1,18 @@
 ;with M as (
 	select Now=GETUTCDATE(), MaxSequence=cast(JSON_VALUE(Value, '$.MaxSequence') as int)
-	from Config 
+	from Config
 	where Instance='MetaStreamSubscription' and Type='CatchupSubscriptionManager'
 ),
 ActivityStreamCommits as (
 	select cast(CommitStamp as datetime) CommitStamp, CheckpointNumber, CommitSequence
 	from Commits
-	where BucketId = 'Meta' and StreamId = '1EE3982777943C1BD64B154C4B5008DC0C3815A0'
```

Comment:
As Pavel mentioned, this should be tested, because without this filters I've seen poor execution  plans on allstate db